### PR TITLE
Solving memory leak when saving a customer

### LINKED ISSUE #70: Memory leak on admin order creation
I haven't looked into what is causing it, but while this module is enabled we are unable to place orders under customer accounts who have thousands of orders because it exceeds the PHP memory limit.

If i find anything i will update.

### PR DESCRIPTION
**Pull request by EmeryEx**
Fixes #70, #87. Customers with a long history of orders can not place orders if their customer account "save" triggers.

### CODE REVIEW

File: src/app/code/community/Zendesk/Zendesk/Model/Observer.php [hunk 1]
```diff
@@ -157,20 +157,24 @@ public function changeIdentity(Varien_Event_Observer $event)
...
         if (is_object($order_totals)) {
             $order_totals
-            ->addFieldToSelect('*')
-            ->addFieldToFilter('customer_id', $customer->getId())
-            ->addFieldToFilter('status', Mage_Sales_Model_Order::STATE_COMPLETE)
-            ->addAttributeToSelect('grand_total')
-            ->getColumnValues('grand_total');
+                ->addFieldToFilter('customer_id', $customer->getId())
+                ->addFieldToFilter('status', Mage_Sales_Model_Order::STATE_COMPLETE)
+            ;
```
Comment (joseconsador) [inline src/app/code/community/Zendesk/Zendesk/Model/Observer.php]: @MarshallHouse  Semi-colon needs to be aligned. 

Can we do this minor change while bumping versions? @jwswj

**Comment by joseconsador** [comment]
Confirmed working, but I had to set memory_limit down to 32M to replicate the issue in #70 . 

### Notes

I have a gist or database available we can use for testing. (adding thousands of orders to a customer)

:+1: 

/cc @miogalang @jwswj @iandjx

**Comment by miogalang** [comment]
:+1:

**Comment by jwswj** [comment]
:+1:

**Pull request merged**